component/app.jsonnet

```
@@ -2,9 +2,11 @@ local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
 local params = inv.parameters.paralus;
 local argocd = import 'lib/argocd.libjsonnet';
+local instance = inv.parameters._instance;
 
-local app = argocd.App('paralus', params.namespace);
+local appName = if instance == 'paralus' then 'paralus' else instance;
+local app = argocd.App(appName, params.namespace);
```

This seems unnecessarily complex. if `instance == 'paralus'` we use hardcoded `'paralus'` but `instance` is `'paralus'` anyway and in all other cases we just use `instance`. So, effectively, the whole `appName` thing doesn't do anything and these two lines are equivalent to:


```suggestion
local app = argocd.App(instance, params.namespace);
```

---

I agree this seems unnecessary, thanks